common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
Some clipping caused by duplicate meshes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120005635-137a4f00-bfe9-11eb-8988-5050c2aecca3.png)

</details>